services/supervisord/supervisord.go

```diff
@@ -406,6 +422,45 @@ func (s *Service) marshalConfig(tmpl *template.Template, settings *models.Settin
 	return b, nil
 }
 
+// addAlertManagerParam parses alertManagerURL
+// and extracts username and password from it to templateParams.
+func addAlertManagerParam(alertManagerURL string, templateParams map[string]interface{}) error {
+	templateParams["AlertmanagerURL"] = alertManagerURL
```

Comment:
TBH this method looks like it can be simplified.

Reply:
Yep, will do it.

Reply:
Was it done?

Reply:
yep it looks simpler.